src/components/pages/TransactionHistoryPage.js

```diff
@@ -40,75 +112,49 @@ const TransactionHistoryPage = () => {
-                  Date
-                </th>
-              </tr>
-            </thead>
-            <tbody className="border border-another-gray border-opacity-30 text-center lg:font-normal sm:font-thin">
-              {transactions.map((transaction, index) => (
-                <tr
-                  key={index}
-                  className={
-                    index % 2 === 0
-                      ? "bg-dark-gray align-left"
-                      : "bg-another-gray align-left"
-                  }
-                >
-                  <td className="lg:px-3 lg:py-5 md:px-2 md:py-2">
-                    {transaction.jobName}
-                  </td>
-                  <td className="lg:px-3 lg:py-5 md:px-2 md:py-2">
-                    {transaction.buyOrSell}
-                  </td>
-                  <td className="lg:px-3 lg:py-5 md:px-2 md:py-2">
-                    {transaction.stockTicker}
-                  </td>
-                  <td className="lg:px-3 lg:py-5 md:px-2 md:py-2">
-                    {transaction.shares}
-                  </td>
-                  <td className="lg:px-3 lg:py-5 md:px-2 md:py-2">
-                    {transaction.amount}
-                  </td>
-                  <td className="lg:px-3 lg:py-5 md:px-2 md:py-2">
-                    {transaction.date}
-                  </td>
-                </tr>
-              ))}
-            </tbody>
-          </table>
-          <div className="p-6 pt-24 pb-20 overflow-auto">
-            <button
-              className="text-white rounded-md bg-another-gray py-2 px-6"
-              onClick={handlePreviousClick}
-            >
-              Previous
-            </button>
-            <button
-              className="text-white rounded-md bg-another-gray py-2 px-6 float-right"
-              onClick={handleNextClick}
-            >
-              Next
-            </button>
-          </div>
+          {isLoading ? (
+            <TableSpinner />
+          ) : (
+            <>
+              <Table data={transactions} header={header}></Table>
+              <div className="p-6 pt-24 pb-20 overflow-auto	">
+                <p className="text-2xl font-light text-center text-white ">
+                  {"Page " + page}
+                </p>
+                {page === 1 ? (
+                  <button
+                    className="text-white rounded-md bg-another-gray py-2 px-6"
+                    disabled
+                  >
+                    Previous
+                  </button>
+                ) : (
+                  <button
+                    className="text-white rounded-md bg-another-gray py-2 px-6"
+                    onClick={handlePreviousClick}
+                  >
+                    Previous
+                  </button>
+                )}
+
+                {transactions.length === 0 || transactions.length !== 10 ? (
```

Comment:
Couldn't this just be if :

`if transactions.length < 10`

Reply:
done